pkgs/by-name/kp/kplex/package.nix

```
@@ -0,0 +1,27 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  lib,
+}:
+stdenv.mkDerivation rec {
+  pname = "kplex";
+  version = "1.4";
+  src = fetchFromGitHub {
+    owner = "stripydog";
+    repo =  "kplex";
+    rev = "v${version}";
+    hash = "sha256-sps9l238hGLJ673kewFH8fOJw0HphEkZbJ+VUIzxC+o=";
+  };
+
+  installPhase = ''
+    install -D -m 0555 kplex $out/bin/kplex
+  '';
+
+  meta = with lib; {
+    description = "A multiplexer for various nmea 0183 interfaces";
+    homepage = "https://www.stripydog.com/kplex/";
```

A changelog is nice to note down, when it exists.

```suggestion
    homepage = "https://www.stripydog.com/kplex/";
    changelog = "https://www.stripydog.com/kplex/changes.html";
```